app/src/main/java/org/p2p/wallet/moonpay/repository/sell/MoonpaySellRemoteRepository.kt

```diff
@@ -46,4 +52,19 @@ class MoonpaySellRemoteRepository(
 
         return ipFlags.isSellAllowed
     }
+
+    @Throws(MoonpaySellError::class)
+    override suspend fun getUserSellTransactions(
+        userAddress: Base58String
+    ): List<MoonpaySellTransaction> = withContext(dispatchers.io) {
```

Comment:
no reason to use dispatchers here, cause we are already in suspend context from retrofit functio